src/arch/host/lib/notifier.c

```diff
@@ -19,7 +19,7 @@ static struct notify *host_notify;
 struct notify **arch_notify_get(void)
 {
 	if (!host_notify)
-		host_notify = rzalloc(RZONE_SYS, SOF_MEM_CAPS_RAM,
+		host_notify = rzalloc(RZONE_SYS, 0, SOF_MEM_CAPS_RAM,
 				      sizeof(*host_notify));
```

Comment:
@tlauda My personal opinion is that adding an extra parameter just for this it doesn't worth the effort. As soon we correctly separate the flags per categories inside the argument we are fine.

Look at gfp_t in Linux kernel. https://lwn.net/Articles/155344/

I don't have any strong preference on this. If others think it is OK we can go with. I would really like to keep things as close as possible to the Linux kernel.

Reply:
I don't like adding arguments also, but having them mixed in 1 variable is even worse.
Ideally it would be kept as it was and be just proxy that calls another function that has that extra argument added.

Reply:
@jajanusz What is the benefit? I understand it would spare me the time to actually change all the calls, but since I've already done it I don't see any point in adding additional proxy methods. Of course I can revert my changes and do that, but it depends whether you're willing to accept this solution or not :)

Reply:
@tlauda I'm ok with what you did, it's improvement anyway. I just wrote about ideal solution - functions that are called often should have as low number of arguments as possible instead of having lengthy calls repeated all the time in code (without mixing multiple arguments into one, cos it's even worse).